2021-04-15 01:07:00,861 - DEBUG - ServerEntity.ServerEntity - [ServerEntity.py:25]: __init__ create entity ServerEntity with id 60772134d164122e74121233 mem_id=2597148334832
2021-04-15 01:07:00,863 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60772134d164122e74121233 
2021-04-15 01:07:00,879 - DEBUG - __main__ - [TcpClient.py:51]: bs: ('battle_0', '127.0.0.1', 8888)
2021-04-15 01:07:00,886 - DEBUG - ServerEntity.Puppet - [ServerEntity.py:25]: __init__ create entity Puppet with id 60772134d164122e74121234 mem_id=2597149033904
2021-04-15 01:07:00,888 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60772134d164122e74121234 
2021-04-15 01:07:27,791 - DEBUG - ServerEntity.ServerEntity - [ServerEntity.py:25]: __init__ create entity ServerEntity with id 6077214fd1641225a8c5a964 mem_id=2536863538144
2021-04-15 01:07:27,793 - DEBUG - server.EntityManager - [EntityManager.py:139]: addentity  entity id  6077214fd1641225a8c5a964 
2021-04-15 01:07:27,809 - ERROR - RpcHandler - [RpcHandler.py:288]: _rpc_reply_fut_obj.rpc_func_name='pick_lowest_load_service_addr': _error="'_asyncio.Future' object is not subscriptable"
2021-04-15 01:07:27,812 - ERROR - __main__ - [TcpClient.py:43]: _err="'_asyncio.Future' object is not subscriptable"
